src/com/company/reverse/Reverse.java

```diff
@@ -0,0 +1,21 @@
+package com.company.reverse;
+
+public class Reverse {
+    public static void main(String[] args) {
+
+
+        System.out.println(reverse("ad12fgiu57"));
+    }
+
+    private static String reverse(String string) {
+        StringBuilder reverseString = new StringBuilder(string.replaceAll("[\\d+]", "")).reverse();
+
+        for (int numOfSymbols   = 0; numOfSymbols < string.length(); numOfSymbols++) {
```

Comment:
in this case name `numOfSymbols` Is not so great.
1. It’s shortened (usually we don’t use short words in names, it’s not always obvious what it means) `numberOfSymbols` should work better
2. This name is a bit ambiguous. Judging only by name `numOfSymbols` I will guess that this variable mean some quantity of symbols. But actually it’s more like position or index..

Reply:
Also formatting is a bit off

Reply:
Done